lib/wallaroo/core/checkpoint/checkpoint_initiator_phase.pony

```diff
@@ -124,7 +124,9 @@ class _CheckpointingPhase is _CheckpointInitiatorPhase
     end
     ifdef debug then
       Invariant(checkpoint_id == _token.id)
-      Invariant(_c_initiator.workers().contains(worker))
+      ifdef "enforce_worker_invariant" then
```

Comment:
Why has this been moved into an ifdef?

Reply:
I would've sworn that that `enforce_worker_invariant ` appeared elsewhere in the code, and that I'd added this one with the same name.  But I can't find another instance.  Hm.

Reply:
Removed by commit f697a511ad108d8a7b7712640558feb0cb30bb3a